### ISSUE TITLE
Add scaffolding for teachers to synthesize questions

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#1815]

**Comment by schanzer** [comment]
@flannery-denny is this different from #2002?

**Comment by flannery-denny** [comment]
Yes. #2002 makes sure that every lesson section has questions in the synthesize. I intend to have it finished by april 15 as it impacts slides.#1815 makes sure that there are answers to all synthesize questions that teachers would need to know answers for. It’s a much more time consuming and less time-sensitive issue.FlanneryOn Apr 2, 2024, at 7:42 PM, Emmanuel Schanzer ***@***.***> wrote:﻿
@flannery-denny is this different from #2002?

**Comment by flannery-denny** [comment]
@schanzer I'm removing the animals starter file from the grouped samples lesson, because it uses the [grouped samples starter file](https://code.pyret.org/editor#share=1Pd8b51rf5A0LD4Zov0HQSFEo3AXICAmo) in the first two lesson sections and I can't come up with a good reason to send them back to the animals starter file in the third lesson section to complete https://www.bootstrapworld.org/materials/fall2025/en-us/lessons/grouped-samples/pages/data-visualizations2.html - revert if you know something I don't.

**Comment by flannery-denny** [comment]
@retabak @schanzer If you have energy, please read over the commits to this issue and make sure my wording works for you. I didn't write most of these questions and am not actually sure what the hoped for responses would be.

**Comment by schanzer** [comment]
@flannery-denny these are incredibly good. Thank you for doing this!

**Comment by flannery-denny** [comment]
@schanzer - Having reviewed all of the Synthesize questions that didn't already have QinA from all of the Algebra and DS lessons this summer, I am
1. Not sure how to answer the questions in the Data Cycle lesson.
2. Think it would be a better use of time to come up with more interesting question than to think about how to answer them. 

Adding their rewrite to #2588

**Comment by schanzer** [comment]
@flannery-denny totally reasonable
